lib/rules/selector-type-no-unknown/README.md

```diff
@@ -8,7 +8,7 @@ Disallow unknown type selectors.
  * This type selector */
 ```
 
-This rule considers tags defined in the HTML and SVG Specifications to be known.
+This rule considers tags which are valid custom elements and those defined in the following Specifications to be known: HTML and SVG.
```

Comment:
This rule considers tags which are defined in the following Specifications to be known: HTML and SVG.